Code-4-Community/hands-across-the-sea-frontend
# Fixes to prepare for demo

## Why
This PR tackles everything that was in the way of successfully getting through the form. Though the form still has lots of bugs, it is usable and you can go through the entire form, through the library report. Most of the data/logic works but there are still some kinks to work out, as well as lots of UI fixes and improvements to make.

I realize this is a large PR. It's unfortunate, but the alternative was having lots of smaller PRs that would've taken lots of approvals (not to mention rebases) to get merged in before Friday, so I judged this to be the better solution. Feel free to skim through the code, but the most important result of this PR is that the form works -- so I'd recommend pulling this branch and testing it locally.

Also, there's lots of testing left to do for the library report stuff I added, but I've ignored that so we can have this ready by Friday.

## This PR
Big features/changes included:
 - Refactored library report ducks so we could share code between yes/no report
 - Added no report page
 - Lots of miscellaneous UI and logic fixes that were preventing normal progression through the form
 - Fixed landing page merge error

## Verification Steps
You should now be able to log in, click Fill Out a Form, and progress through the entire form until the end (currently the last page is the library report form). Most of the submissions work, some don't or have bugs.
